src/components/templates/PartyMap/components/RoomModalOngoingEvent/RoomModalOngoingEvent.tsx

```diff
@@ -82,8 +75,8 @@ export const RoomModalOngoingEvent: React.FC<RoomModalOngoingEventProps> = ({
         onMouseOver={triggerAttendance}
         onMouseOut={clearAttendance}
         className="btn btn-primary room-entry-button"
-        onClick={joinRoom}
-        id={`enter-room-in-ongoing-event-card-${room.title}`}
+        onClick={onRoomEnter}
+        id={`enter-room-in-ongoing-event-card-${roomTitle}`}
```

Comment:
In react, we pretty much never have a need for using `id`, so this can probably be removed altogether.